calico/felix/splitter.py

```diff
@@ -44,6 +48,12 @@ def apply_snapshot(self, rules_by_prof_id, tags_by_prof_id,
         """
         Replaces the whole cache state with the input.  Applies deltas vs the
         current active state.
```

Comment:
Well done documenting what these are, but I'd like more.

* Is it conventional that mapping => dict? If so, fine. If not, please replace mapping with dict. 
* We should probably specify what a rule is here.

I'm thinking something like :
``:param rules_by_prof_id: A dict mapping security profile ID to a list of rules, each of which is a dict``